nirum.cabal

```diff
@@ -135,6 +144,7 @@ test-suite spec
                ,       nirum
                ,       parsec
                        -- only for dealing with htoml's ParserError
+               ,       pretty                   >=1.1.3   && <2
```

Comment:
As `aeson`,`aeson-pretty` and `pretty` are common dependencies to `library` and `test-suite spec`, it'd better omit version specifies here.